From edcb4ae97bf2ac22f9d8c01a6989dabbc24c058c Mon Sep 17 00:00:00 2001
From: Qingqing Zhou <quic_qqzhou@quicinc.com>
Date: Tue, 24 Sep 2024 17:15:16 +0800
Subject: [PATCH 08/66] dt-bindings: firmware: qcom,scm: document SCM on QCS615
 SoCs

Document scm compatible for QCS615 SoCs. It is an interface to
communicate to the secure firmware.

Signed-off-by: Qingqing Zhou <quic_qqzhou@quicinc.com>
---
 Documentation/devicetree/bindings/firmware/qcom,scm.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/firmware/qcom,scm.yaml b/Documentation/devicetree/bindings/firmware/qcom,scm.yaml
index 1a6278e5571b..63bac81f211a 100644
--- a/Documentation/devicetree/bindings/firmware/qcom,scm.yaml
+++ b/Documentation/devicetree/bindings/firmware/qcom,scm.yaml
@@ -66,6 +66,7 @@ properties:
           - qcom,scm-sm8550
           - qcom,scm-sm8650
           - qcom,scm-qcs404
+          - qcom,scm-qcs615
           - qcom,scm-x1e80100
       - const: qcom,scm
 
-- 
2.34.1

